spec/migrate_password/001_tables.rb

```diff
@@ -47,7 +53,7 @@
       run "REVOKE ALL ON FUNCTION rodauth_previous_password_hash_match(int8, text) FROM public"
       run "GRANT INSERT, UPDATE, DELETE ON account_previous_password_hashes TO #{user}"
       run "GRANT SELECT(id, account_id) ON account_previous_password_hashes TO #{user}"
-      run "GRANT USAGE ON account_previous_password_hashes_id_seq TO #{user}"
+      # run "GRANT USAGE ON account_previous_password_hashes_id_seq TO #{user}"
```

Comment:
It seems odd to just comment this out.  Shouldn't it be conditional (run for Bigint, not run for uuid)?

Reply:
This was accidentally added to the patch while we were working out how to support UUIDs with that table.

In the final implementation `account_previous_password_hashes` always uses a `int8` primary key, therefore the password user should always be granted usage on `account_previous_password_hashes_id_seq`.

I've pushed a patch to uncomment the line.